content/learn/getting-started/npm.fr.md

```diff
@@ -0,0 +1,133 @@
+authors: flaviocopes, MylesBorins, LaRuaNa, jgb-solutions, amiller-gh, ahmadawais, AugustinMaurou
+category: learn
+---
+
+<Alert>Gardez à l'esprit qu'il s'agit d'une brève introduction à <code>NPM</code>. Pour un guide plus approfondi, consultez le <a href="https://docs.npmjs.com/">NPM documentation</a>.</Alert>
+
+## Introduction à npm
+
+`npm` est le gestionnaire de paquets standard pour Node.js.
+
+En septembre 2022, plus de 2,1 millions de paquets ont été répertoriés dans le registre npm, ce qui en fait le plus grand dépôt de code monolingue sur Terre, et vous pouvez être sûr qu'il y a un paquet pour (presque !) tout.
+
+Il a commencé comme un moyen de télécharger et de gérer les dépendances des paquets Node.js, mais il est depuis devenu un outil utilisé également dans le JavaScript frontal.
+
+> [**Yarn**](https://yarnpkg.com/en/) et [**pnpm**](https://pnpm.io) sont des alternatives à npm cli. Vous pouvez également les consulter.
+
+## Paquets
+
+`npm` gère les téléchargements des dépendances de votre projet.
+
+### Installer toutes les dépendances
+
+Si un projet a un fichier `package.json`, en exécutant
+
+```bash
+npm install
+```
+
+il installera tout ce dont le projet a besoin, dans le dossier `node_modules`, en le créant s'il n'existe pas déjà.
+
+### Installation d'un seul paquetage
+
+You can also install a specific package by running
+
+```bash
+npm install <package-name>
+```
+
+De plus, depuis npm 5, cette commande ajoute `<package-name>` au fichier `package.json` _dependencies_. Avant la version 5, vous deviez ajouter le flag `--save`.
+
+Vous verrez souvent d'autres drapeaux ajoutés à cette commande :
+
+- `--save-dev` installe et ajoute l'entrée au fichier `package.json` _devDependencies_.
+- `--no-save` installe mais n'ajoute pas l'entrée au fichier `package.json` _dependencies_.
+- `--save-optional` installe et ajoute l'entrée au fichier `package.json` _optionalDependencies_.
+- `--no-optional` empêche l'installation des dépendances optionnelles.
+
+Des raccourcis des drapeaux peuvent également être utilisés :
+
+- \-S : `--save`
+- \-D : `--save-dev`
+- \-O : `--save-optional` (enregistrement facultatif)
+
+La différence entre _devDependencies_ et _dependencies_ est que le premier contient des outils de développement, comme une bibliothèque de test, tandis que le second est fourni avec l'application en production.
+
+Quant à _optionalDependencies_, la différence est que l'échec de la construction de la dépendance ne fera pas échouer l'installation. Mais c'est à votre programme qu'il incombe de gérer l'absence de cette dépendance. En savoir plus sur [optional dependencies](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#optionaldependencies).
```

Comment:
Maybe also translate the link of `[optional dependencies]` by `[dépendances optionnelles]`